gulpfile.ts

```diff
@@ -140,11 +160,16 @@ gulp.task('publish', () => {
               upToDatePackages++;
             }
             else {
-              _logger.log(`Publishing package "${packageName}" with version "${localPackageVersion}"...${args.whatif ? " (SKIPPED)" : ""}`);
+              _logger.log(`Packing package "${packageName}" with version "${localPackageVersion}"...${args.whatif ? " (SKIPPED)" : ""}`);
               if (!args.whatif) {
                 try {
                   npmInstall(packageFolderPath);
-                  execSync(`npm publish`, { cwd: packageFolderPath });
+                  execSync("npm install", { cwd: packageFolderPath });
```

Comment:
Shouldn't the prepack work you did recently do `npm install` automatically?

Reply:
@danschultequb new alias? yes, I will remove `npm install` as soon as we regenerate all packages using the new generator.

Reply:
Sorry, I forgot that I was logged in with my personal GitHub account. I'll still be using @daschult for all work related stuff.